### ISSUE TITLE
`HIP` instead of `HIS`

### ISSUE THREAD

**Issue by joaomcteixeira** [julie-forman-kay-lab/IDPConformerGenerator#234]
IDPCG is generating `HIP` residues by default instead of `HIS`, and even if `-dsd` is given. A user has reported.

@menoliu can you help me here to sort this out? It should be `HIS` unless `HIP`is provided in the fasta with the letter `p`.

Is there any reason why this is like this that I may not be remembering?

Thanks so much! If you can, please give priority to this issue.

**Comment by menoliu** [comment]
Yes I have realized that but I assumed it was intentional to accommodate double protonation? I should have asked what you meant here in `libparse.py` lines `372-382`:
```
# TODO: correct for HIS/HIE/HID/HIP
def translate_seq_to_3l(input_seq, histidine_protonation="HIS"):
    """
    Translate 1-letter sequence to 3-letter sequence.

    Currently translates 'H' to 'HIP', to accommodate double protonation.
    """
    return [
        histidine_protonation if _res == 'H' else aa1to3[_res]
        for _res in input_seq
        ]
```
